examples/transforms/extract-field-transform.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# This script will start Conduit and set up a pipeline which demonstrates
+# a transform replacing a record's structured payload with one of its fields.
+
+# The pipeline consists of:
+# * a generator source, generating structured records
+# * a file destination (it writes to file_destination.txt)
+# * a processor with the built-in extract-field transform.
+
+echo "Running Conduit..."
+docker run --rm  --name conduit -p 8080:8080 -d ghcr.io/conduitio/conduit:latest
+# todo: implement healthcheck in docker
+sleep 2
+
+echo "Creating pipeline..."
+PIPELINE_ID=$(
+curl -Ss -X POST 'http://localhost:8080/v1/pipelines' -d '
+{
+    "config":
+    {
+        "name": "test-pipeline-builtin-transform",
+        "description": "Test pipeline with built-in transform"
+    }
+}' | jq -r '.id'
+)
+
+
+echo "Creating a generator source..."
+SOURCE_CONN_REQ=$(
+jq -n --arg pipeline_id "$PIPELINE_ID" '{
+    "type": "TYPE_SOURCE",
+    "plugin": "builtin:generator",
+    "pipeline_id": $pipeline_id,
+    "config":
+    {
+        "name": "my-generator-source",
+        "settings":
+        {
+            "fields": "id:int,name:string,company:string,trial:bool",
+            "format": "structured",
+            "readTime": "500ms",
+            "recordCount": "1"
+        }
+    }
+}'
+)
+CONNECTOR_ID=$(curl -Ss -X POST 'http://localhost:8080/v1/connectors' -d "$SOURCE_CONN_REQ" | jq -r '.id')
+
+echo "Creating a built-in transform..."
+TRANSFORM_REQ=$(
+jq -n --arg connector_id "$CONNECTOR_ID" '{
+     "name": "extractfieldpayload",
+     "type": 1,
+     "parent":
+     {
+         "type": 1,
+         "id": $connector_id
+     },
+     "config":
+     {
+         "settings":
+         {
+             "field": "name"
+         }
+     }
+ }'
+)
+curl -Ss -X POST 'http://localhost:8080/v1/processors' -d "$TRANSFORM_REQ" > /dev/null
+
+echo > file_destination.txt
```

Comment:
This creates the file on the local disk, but Conduit runs in a container that doesn't have access to this file. When running the example it fails with this error:
```
node a2f165a0-34ff-495a-8d94-b500238320f4 stopped with error: could not open destination connector: open /Users/lovro/dev/conduitio/conduit/examples/transforms/file_destination.txt: no such file or directory
```

Reply:
Last minute refactoring...: ) Fixed.